okcurl/okcurl

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+../gradlew -q --console plain installDist
```

Comment:
What does this do?

Reply:
silently builds (with gradle caching etc), so that you can run "./okcurl https://..." from a git checkout and it *just* works